components/skeleton/nz-skeleton.component.html

```diff
@@ -0,0 +1,18 @@
+<ng-container *ngIf="nzLoading">
+  <div class="ant-skeleton-header">
+    <span *ngIf="!!nzAvatar" 
+      class="ant-skeleton-avatar" 
+      [ngClass]="avartarClassMap">
+    </span>
+  </div>
+  <div class="ant-skeleton-content">
+    <h3 *ngIf="!!nzTitle" class="ant-skeleton-title" [style.width]="toCSSUnit(_title.width)"></h3>
```

Comment:
It's not a good idea to access a private variable in template files. It may cause errors in `--aot` compiling.